app/Http/Controllers/Auth/AuthController.php

```diff
@@ -38,40 +36,38 @@ public function getCheckpoint()
 
     public function getIndex()
     {
+        $state = str_random(5);
```

Comment:
no need. the goal of te state here is not for uniqueness (sessions are already unique). It's goal is to ensure the auth url is always fresh and used once (per session)